README.md

```diff
@@ -1,10 +1,23 @@
-# Vividus starter
+# Vividus Starter
 
-Vividus starter purpose is to help you start developing automation Vividus Framework;
+The puspose of this repository is to help users start creating automated tests using Vividus.
 
+# How to run the existing tests
 1. Clone the repository: `git clone --recursive https://github.com/vividus-framework/vividus-starter.git`
-2. Execute command: `cd vividus-starter`
-3. Execute command `./gradlew runStories` to run your tests;
-4. Execute command `./gradlew build eclipse` or `./gradlew build idea` to build for your favorite IDE;
-5. Import project into IDE;
-6. Have fun !!!
+1. Execute command: `cd vividus-starter`
+1. Execute command `./gradlew runStories` to run your tests
+
+# How to create own test project using starter
+1. Login to Github: http://github.com. (Create a new account before if you don't have any)
+1. Generate your own project based on the starter project:
+    - Open generator page: https://github.com/vividus-framework/vividus-starter/generate
+    - Provide repository name for your test project
+    - Choose option to make your repository `Public` or `Private` (`Private` is recommended from start, it can be changed in future)
+1. Clone your newly generated repository: `git clone --recursive <generatedProjectLink>.git`
```

Comment:
please add cd before

Reply:
`cd` where?

Reply:
before cloning bs, imho it's more understandable to express it as command than mentioning it the last line of the instruction + not anyone knows how to change directory in command line